corehq/form_processor/models.py

```diff
@@ -755,6 +739,10 @@ def get_obj_by_id(cls, case_id):
         from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
         return CaseAccessorSQL.get_case(case_id)
 
+    @property
+    def default_parent_identifier(self):
+        return CommCareCaseIndexSQL.PARENT_IDENTIFIER
```

Comment:
i was just trying to have one way to reference the constant. should i just add the constant to abstract_models.py at the module level and reference that everywhere?